web_responsive/static/src/css/web_responsive.scss

```diff
@@ -31,8 +54,7 @@
             }
 
             .o_menu_sections,
-            .o_menu_systray,
-            {
+            .o_menu_systray {
```

Comment:
I'd leave this as it was, it looks good for when there are multiple selectors IMHO.

The same applies for similar diffs.

Reply:
I changed it because the linter doesn't like the last comma, and change the Allman brace style to K&R style to be like as the rest of the file:
![screenshot_20190612-120115_1920x1080](https://user-images.githubusercontent.com/731270/59342404-04ba0500-8d0a-11e9-9bb1-d591191e1a3f.png)

Reply:
Hmm but it's valid syntax AFAIK...

Honestly we don't have a linter for *CSS right now at @OCA. It was just a matter of taste. As we usually leave the trailing comma wherever possible to reduce future diffs, I was doing the same here.

I wouldn't introduce these kind of diffs until at least we have a linter here, but I'm open. What do you think @pedrobaeza @moylop260?

Reply:
Let's minimize now the diff and open a discussion in MQT about the CSS/SCSS linter.

Reply:
Any way, @Tardo, any lint fixing is convenient to be put in a separate commit for isolate this noise from the rest of the changes.